api/v1/controllers/EventController.js

```diff
@@ -69,11 +79,11 @@ router.use(bodyParser.json());
 router.use(middleware.auth);
 
 router.post('/', middleware.request(requests.EventCreationRequest), middleware.permission(roles.ORGANIZERS), createEvent);
-router.get('/', getAllEvents);
```

Comment:
Why are we getting rid of the ability to get all events? I feel like a URL parameter (e.g. `?active=1`) that specifies active-only would be better than removing the ability to get past events.